.travis.yml

```diff
@@ -6,22 +6,20 @@ php:
     - 5.4
     - 5.5
     - 5.6
+    - 7.0
     - hhvm
-    - hhvm-nightly
 
 env:
     global:
         - COMPOSER_PREFER_LOWEST=false
+        - SYMFONY_DEPRECATIONS_HELPER=weak
         - SYMFONY_VERSION=2.3.*
-
-cache:
-    directories:
-        - vendor
-        - $HOME/.composer/cache
+        - SYMFONY_PHPUNIT_BRIDGE_VERSION=2.7.*
 
 install:
     - composer self-update
     - composer require --no-update symfony/form:${SYMFONY_VERSION}
+    - composer require --dev --no-update symfony/phpunit-bridge:${SYMFONY_PHPUNIT_BRIDGE_VERSION}
```

Comment:
Why not adding this to the `composer.json`? It's also useful when running tests locally.